content/courses/native-onchain-development/hello-world-program.md

```diff
@@ -317,78 +312,76 @@ use solana_program::{
 entrypoint!(process_instruction);
 
 pub fn process_instruction(
-    program_id: &Pubkey,
-    accounts: &[AccountInfo],
-    instruction_data: &[u8]
-) -> ProgramResult{
+    _program_id: &Pubkey,
+    _accounts: &[AccountInfo],
+    _instruction_data: &[u8],
+) -> ProgramResult {
     msg!("Hello, world!");
 
     Ok(())
 }
 ```
 
-#### 4. Build and Deploy
+### 4. Build and Deploy
 
-Now let's build and deploy our program using Solana Playground.
+Now, let's build and deploy our program using Solana Playground.
 
 ![Gif Solana Playground Build and Deploy](/public/assets/courses/unboxed/hello-world-build-deploy.gif)
 
-#### 5. Invoke Program
+### 5. Invoke Program
 
-Finally, let's invoke our program from the client side. The focus of this lesson
-is to build our Solana program, so we’ve gone ahead and provided
+Finally, let's invoke our program from the client side. The main focus of this
+lesson is building our Solana program, so we've provided
 [the client code to invoke our “Hello, world!” program](https://github.com/Unboxed-Software/solana-hello-world-client)
 for you to download.
 
-The code provided includes a `sayHello` helper function that builds and submits
-our transaction. We then call `sayHello` in the main function and print a Solana
-Explorer URL to view our transaction details in the browser.
-
-Open the `index.ts` file you should see a variable named `programId`. Go ahead
-and update this with the program ID of the “Hello, world!" program you just
-deployed using Solana Playground.
+This code includes a `sayHello` helper function that constructs and submits the
+transaction. In the `index.ts` file, you'll find a variable named `programId`.
+Update this with the program ID of the “Hello, world!” program you just deployed
+using Solana Playground.
 
 ```typescript
 let programId = new web3.PublicKey("<YOUR_PROGRAM_ID>");
 ```
 
-You can locate the program ID on Solana Playground referencing the image below.
+You can find the program ID on Solana Playground as shown below.
 
 ![Gif Solana Playground Program ID](/public/assets/courses/unboxed/hello-world-program-id.gif)
```

Comment:
You might want to make a new version of this image (or an mp4) if Solana Playground looks out of date. You decide.

Reply:
I have not updated the new version as there is not much difference in UI and UX.